web/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="stylesheet" href="style/createProject.css">
     <link rel="stylesheet" href="https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css">
     <link rel="stylesheet" href="style/fonts/font-awesome/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
If I use the `v4-dev` version, the behavior of the site seams to be a little bit strange. Maybe because its the `dev` version. If I use just the `3.3.7` version everything works fine.